inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/view/InAppMessageBaseView.kt

```diff
@@ -193,13 +193,17 @@ internal open class InAppMessageBaseView(context: Context, attrs: AttributeSet?)
         }
 
         buttonView.backgroundTintList = ColorStateList.valueOf(bgColor)
-        // Button stroke color equals to button text color.
-        buttonView.strokeColor = ColorStateList.valueOf(textColor)
+        buttonView.strokeColor = resources.getColorStateList(R.color.modal_border_color_light_grey, context.theme)
```

Comment:
Why are we using light grey?

Reply:
That's the color I got from Figma designs